### PR TITLE
Update RF13

### PR DESCRIPTION
**Pull request by tmieslinger**
Based on Lutz changes to the YAML files I additionally added the new dropsonde definitions and adjusted the `cricle` and `circle break` times and slightly extended the start/end times of the `lidar leg`. 
I suggest to merge this branch and close the former one by Lutz.

### CODE REVIEW

File: flight_phase_files/HALO_RF13_20200213_info.yaml [hunk 1]
```diff
@@ -25,59 +30,326 @@ remarks:
...
+    - HALO-0213_s72
+    BAD: []
+    UGLY:
+    - HALO-0213_s73
+- kinds:
+  - lidar_leg
+  - straight_leg
+  name: lidar leg
+  irregularities: []
+  segment_id: HALO-0213_lc1
```
Comment (marc-prange) [inline flight_phase_files/HALO_RF13_20200213_info.yaml]: update to "HALO-0213_ll1"

File: flight_phase_files/HALO_RF13_20200213_info.yaml [hunk 2]
```diff
@@ -25,59 +30,326 @@ remarks:
...
+  - circle_break
+  name: circle break 1
+  irregularities: []
+  segment_id: HALO-0213_cb1
+  start: 2020-02-13 09:15:39
+  end: 2020-02-13 09:28:31
+  dropsondes:
+    GOOD: []
+    BAD:
+    - HALO-0213_s13
```
Comment (marc-prange) [inline flight_phase_files/HALO_RF13_20200213_info.yaml]: After discussion of Geet and Tobi on dropsondes in circle breaks, it seems consequent to also move this BAD sonde to the circle 2.

File: flight_phase_files/HALO_RF13_20200213_info.yaml [hunk 3]
```diff
@@ -25,59 +30,326 @@ remarks:
...
+    UGLY:
+    - HALO-0213_s19
+- kinds:
+  - circle
+  name: circle 1
+  irregularities: []
+  segment_id: HALO-0213_c1
+  start: 2020-02-13 08:19:47
+  end: 2020-02-13 09:15:39
```
Comment (marc-prange) [inline flight_phase_files/HALO_RF13_20200213_info.yaml]: One second back to avoid overlap.

**Review by marc-prange — COMMENTED** [review]
Looks very good, 3 minor points. 👍 
Thanks a lot Theresa for the update!

File: flight_phase_files/HALO_RF13_20200213_info.yaml (hunk 1 above)
Comment (tmieslinger) [reply flight_phase_files/HALO_RF13_20200213_info.yaml]: Absolutely right!

File: flight_phase_files/HALO_RF13_20200213_info.yaml (hunk 2 above)
Comment (tmieslinger) [reply flight_phase_files/HALO_RF13_20200213_info.yaml]: I moved the bad sonde from the circle break to circle 2 and started the circle 60s before that sonde.

**Pull request merged**